.github/workflows/ci.yaml

```diff
@@ -60,9 +59,9 @@ jobs:
     runs-on : ${{ matrix.os }}
```

Comment:
Looks like we still need to update these other `matrix` references to use the new keys. So any `matrix.os` should end up being `matrix.arch`.

Reply:
I think we only need to change the name of the artifact to be unique for each arch. That's what I understood from https://github.com/actions/upload-artifact/issues/478 and have made a change accordingly. Let's see. 🤞🏽